checker/bin-devel/build.sh

```
@@ -1,19 +1,14 @@
 #!/bin/bash
 
-echo Deprecated: use ./gradlew assemble instead.
 echo Entering checker/bin-devel/build.sh in "$(pwd)"
+echo checker/bin-devel/build.sh is deprecated: use ./gradlew assemble instead.
 
 # Fail the whole script if any command fails
 set -e
 
 SCRIPT_DIR="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 source "$SCRIPT_DIR"/clone-related.sh
```

Shellcheck (0.11.0)

[info] Not following: ./clone-related.sh was not specified as input (see shellcheck -x).

(SC1091)